code/AEPSDK.brs

```diff
@@ -275,6 +275,45 @@ function AdobeAEPSDKInit(taskNode = invalid as dynamic) as object
             m._private.dispatchEvent(event)
         end function,
 
+        ' ****************************************************************************************************
+        '
+        ' Returns the callback with data as ECID (Experience Cloud ID) persisted by the Edge SDK.
+        ' NOTE: This API will generate a new ECID if no existing ECID is found in persistent storage.
+        '
+        ' @param callback as function(context, ecid) : handle ECID response
+        ' @param context as dynamic : context to be passed to the callback function
+        '
+        ' ****************************************************************************************************
+
+        getExperienceCloudId: function(callback = _adb_defaultCallback as function, context = invalid as dynamic) as void
+            _adb_logDebug("API: getExperienceCloudId()")
+            if callback = invalid
+                _adb_logError("setExperienceCloudId() - Cannot get ECID, invalid callback passed.")
+                return
+            end if
+
+            if context = invalid
+                _adb_logError("setExperienceCloudId() - Cannot get ECID, invalid context passed.")
```

Comment:
The `context` parameter is optional, can we change it to a debug log?

Reply:
I removed the check and the log since its optional